### PR TITLE
OcAppleKernelLib: Add new patch for macOS 13.3+ IOAHCIBlockStorage patch, by @vit9696

### PR DESCRIPTION
**Pull request by PMheart**
This closes https://github.com/acidanthera/bugtracker/issues/2256

@Andrey1970AppleLife This is the integrated version of https://github.com/acidanthera/bugtracker/issues/2256#issuecomment-1487606119. Could you please test and confirm if it works? Thanks.

### CODE REVIEW

File: Library/OcAppleKernelLib/CommonPatches.c [hunk 1]
```diff
@@ -856,6 +856,78 @@ PATCHER_GENERIC_PATCH
...
+STATIC
+CONST UINT8
+  mIOAHCIBlockStoragePatch133Find2[] = {
+  0x48, 0x8D, 0x3D, 0x00, 0x00, 0x00, 0x00,
+  0xBA, 0x05, 0x00, 0x00, 0x00
+};
+
+STATIC
+CONST UINT8
+  mIOAHCIBlockStoragePatch133Replace2[] = {
```
Comment (vit9696) [inline Library/OcAppleKernelLib/CommonPatches.c:911]: Replace is also the same, could keep.

**Comment by Andrey1970AppleLife** [comment]
Doesn't work. With patches from @vit9696 works.

**Comment by PMheart** [comment]
Could you please provide any logs? @Andrey1970AppleLife

**Comment by PMheart** [comment]
I don’t see any logging for ThirdPartyDrives. Are you sure that it is turned on?

**Comment by PMheart** [comment]
@Andrey1970AppleLife I see the problem now. Sorry and thanks! Please try the latest force-pushed commit; it should work now.

**Comment by Andrey1970AppleLife** [comment]
Works

**Pull request merged**